js/src/components/campaign-conversion-notice/conversion-statuses.js

```diff
@@ -58,8 +57,7 @@ const AFTER_CONVERSION = {
 		}
 	),
 	externalLink: {
-		//TODO Update link when it is defined
-		link: '#',
+		link: 'https://support.google.com/google-ads/answer/11576060',
```

Comment:
💅  This link could be stored as a constant to avoid repetition

Reply:
Added here: [18c8557](https://github.com/woocommerce/google-listings-and-ads/pull/1515/commits/18c8557804b571d5eba6867e440689eab03f1d59)